components/scream/src/physics/shoc/CMakeLists.txt

```diff
@@ -112,6 +124,8 @@ foreach (SHOC_LIB IN LISTS SHOC_LIBS)
     ${CMAKE_CURRENT_BINARY_DIR}/${SHOC_LIB}_modules
   )
   target_link_libraries(${SHOC_LIB} physics_share scream_share)
+  target_compile_options(${SHOC_LIB} PUBLIC $<$<COMPILE_LANGUAGE:Fortran>:${SCREAM_Fortran_FLAGS}>
+                                     PRIVATE $<$<COMPILE_LANGUAGE:CXX>:${SCREAM_CXX_FLAGS} ${SHOC_HIP_FLAGS}>)
```

Comment:
I don't think we have `SCREAM_CXX_FLAGS` defined anywhere. You can remove it.

Reply:
thanks, i removed that line completely, it was only there for O0 hack.